### ISSUE TITLE
Overview still shows byteStride as part of accessor

### ISSUE THREAD

**Issue by FlyingGaz** [KhronosGroup/glTF#1130]
The glTF 2.0 [overview][1] still shows byteStride as part of accessor.
With version 2.0 this property was moved to bufferView.

[1]: https://github.com/KhronosGroup/glTF/blob/master/specification/2.0/figures/gltfOverview-2.0.0.png

**Comment by javagl** [comment]
This would actually be an issue for the https://github.com/javagl/gltfOverview/ repo. In any case, this has been fixed in https://github.com/javagl/gltfOverview/commit/1297fcb7f33d0bc98653e42433ff1b20cc478b54 , but there hasn't been a release in the meantime.

I'll prepare a "2.0.0a" release ASAP (leaving this one open until it's done)

**Comment by javagl** [comment]
Does one of the Git(Hub) wizards know how to cherry-pick https://github.com/javagl/glTF/commit/735fe9f31368714a90d1f06856d856140af9ed6b ? (Otherwise, I'd try it with a branch)

**Comment by emackey** [comment]
Actually let's not use my cherry-pick.  @javagl you have another PR #1136 already open that is on the same branch, so the overview got added there automatically.

**Comment by emackey** [comment]
Fixed by merging #1136.

**Comment by javagl** [comment]
> you have another PR #1136 already open

Yes, that was the main problem, but I noticed it only after updating the overview link (and fixing stuff in Git for me usually just boils down to https://xkcd.com/1597/ ...)

Thanks for taking care of this!

**Comment by emackey** [comment]
Sure.  Our usual practice (on Cesium and related projects) is to not push any changes to `master` even on a fork.  Always create a new branch off the upstream master, named after the feature or fix (such as `overview-2.0.0a` or some such), commit and push the changes from there.  That way you can have multiple PRs open at once from different branches and not get the wires crossed.

**Comment by javagl** [comment]
That's certainly a far better practice and avoids messing up the history like in this case. 

(As a lame excuse: Most of my personal projects are not forks and have a rather linear history. For *really* collaborative projects like the `glTF*` ones, I'll try to adopt this branching approach in the future)

Thanks again.
